index.js

```diff
@@ -18,15 +22,9 @@ module.exports = function(source) {
   options.dest = options.dest || '';
   options.filename = options.filename || this.resourcePath;
   options.Evaluator = CachedPathEvaluator;
-  // Keep track of webpack-resolved imports.
-  options.resolvedImports = {};
 
-  var dependencies = {};
-  options.addDependencies = function(filenames) {
-    filenames.forEach(function(filename) {
-      dependencies[filename] = true;
-    });
-  };
+  // Attach `importCache` to `options` so that the `Evaluator` can access it.
+  var importCache = options.importCache = new ImportCache(this);
```

Comment:
All the code that was removed in this file was moved to this `ImportCache` class.